### PR TITLE
 (Chore) : Upgrade all Package Dependency , FIX GULP-SaSS issue. Proj…

### PR DESCRIPTION
**Pull request by naime-hossain**
1. Update All Dependency 
2. 2. Fix Depcracation error while run yarn / npm install
3. fix gulp-sass issue

### CODE REVIEW

**Comment by naime-hossain** [comment]
@moesaid can you check the pull request

**Comment by moesaid** [comment]
> @moesaid can you check the pull request

Thanks for your time!!! i will make sure to review the PR this weekend! thanks again Hossain!

**Comment by alexis-coulombe** [comment]
Any news @moesaid  ? 

Thanks for his work, i successfully installed the project using his repository.

**Comment by naime-hossain** [comment]
@moesaid can you check the PR  and merge? Thanks

File: tailwind.config.js
Comment (moesaid) [inline tailwind.config.js:1]: @ naime-hossain Hey bud ! Sorry for being out of sync here. Could you explain why removing the customized config and going pair metal?

**Review by moesaid — CHANGES_REQUESTED** [review]

File: tailwind.config.js
Comment (moesaid) [reply tailwind.config.js:1]: never mind, I gotcha!

**Pull request merged**